docs_CN/Endpoints.md

```diff
@@ -113,9 +116,13 @@ let provider = MoyaProvider<GitHub>(requestClosure: requestClosure)
 
 ```swift
 { (endpoint: Endpoint<ArtsyAPI>, done: MoyaProvider.RequestResultClosure) in
-    var request: URLRequest = endpoint.urlRequest
-    request.httpShouldHandleCookies = false
-    done(.success(request))
+	do {
```

Comment:
This is very nitty but mind shifting this ident over <- ?

Reply:
yes, you are right, i have already push a new commit.